crates/turbopack-ecmascript/js/src/runtime.dom.js

```diff
@@ -0,0 +1,34 @@
+/** @typedef {import('../types/backend').RuntimeBackend} RuntimeBackend */
+
+/** @type {RuntimeBackend} */
+const BACKEND = {
+  loadChunk(chunkPath, _from, onLoad, onError) {
+    if (chunkPath.endsWith(".css")) {
+      const link = document.createElement("link");
+      link.rel = "stylesheet";
+      link.href = `/${chunkPath}`;
+      link.onerror = () => {
+        onError();
+      };
+      link.onload = () => {
+        // CSS chunks do not register themselves, and as such must be marked as
+        // loaded instantly.
+        onLoad();
+      };
+      document.body.appendChild(link);
+    } else if (chunkPath.endsWith(".js")) {
+      const script = document.createElement("script");
+      script.src = `/${chunkPath}`;
+      // We'll only mark the chunk as loaded once the script has been executed,
+      // which happens in `registerChunk`.
```

Comment:
That's another story, but we should handle `script.onload` too. If no chunk registration happened before that, that is an error.